project.md

```diff
@@ -11,41 +11,79 @@ _What need is met by your idea? Who are you helping? What is that person's
+
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+
+
+While other DSL’s such as Icalendar exist, it can be difficult to edit or make them by just using text files. The default way to edit these is to use a calendar app or website, which can be slower and more difficult to use. By creating a DSL, many of the unnecessary features and bloat can be removed, limiting the complexity. This should result in a simple to use, easy to understand language that can be used by anyone to quickly create schedules that can then be previewed or exported to other formats. I also hope that by giving the user the option to use syntax inspired by natural language or by using syntax that creates a schedule as quick as possible, the language will be useful by both people with limited experience in DSL’s and power users who have experience in Computer Science and DSL’s.
+
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+
+I originally came up with the idea out of frustration for Google Calendar. I have terrible organization/time management, so a schedule is very important to me. However, I don’t want an overly complicated schedule that’s very detailed because then it will be difficult for me to pay attention to it or make changes. In a sense, I would be putting more effort into maintaining the calendar than I would be in attending the events on the calendar. However, the options given by Google Calendar can sometimes slow down the user and interacting with the calendar can be very slow. I wanted an option to just quickly type out an event name, start/end time, days it occurs on, and a description if it’s needed. 
+	This led me to the idea of making a DSL by limiting the features a lot of calendar apps have, and by doing so creating a robust DSL that could quickly create new schedules and events. I’m ultimately excited about this idea because I personally would use it. I think it would help me be more organized and manage my time better, and if there are people who feel similarly with their calendar app, it would be amazing if this could help them make schedules as well.
+
```

Comment:
I think this is an interesting idea because most DSLs for calendar's and scheduling Want to have a bunch of features - the one that comes to mind that you haven't mentioned is Notion, which I use for it's calendar and planning capabilities - but also definitely has a lot of "extra" stuff if was trying to primarily be a scheduler. It hadn't occurred to me that there are people out there who don't want all the nice features of google meets, etc. so I'm interested to see how the idea gets implemented.